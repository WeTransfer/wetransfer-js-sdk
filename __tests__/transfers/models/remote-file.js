const { RemoteFile } = require('../../../src/transfers/models');

describe('RemoteFile model', () => {
  let file = null;
  beforeEach(() => {
    file = new RemoteFile({
      id: 'random-hash',
      multipart: {
        parts_numbers: 3,
        chunk_size: 195906,
      },
      name: 'kittie.gif',
      size: 195906,
      type: 'file',
    });
  });

  describe('contructor', () => {
    it('should create a normalized file', () => {
      expect(file).toMatchSnapshot();
    });
  });
});
