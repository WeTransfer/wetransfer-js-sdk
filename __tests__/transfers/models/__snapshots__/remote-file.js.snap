// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoteFile model contructor should create a normalized file 1`] = `
Object {
  "id": "random-hash",
  "multipart": Object {
    "chunk_size": 195906,
    "parts_numbers": 3,
  },
  "name": "kittie.gif",
  "size": 195906,
  "type": "file",
}
`;
